compiler/testData/debug/stepping/voidLambdaStepInline.kt

```diff
@@ -0,0 +1,20 @@
+
+
+
+
+// FILE: test.kt
+fun box(): String {
+    run { println("OK") }
```

Comment:
I believe that we normally avoid having println calls in tests to not generate stdout output. Does this should the same if you call some other function?

Reply:
Not quite the same, but it still illustrates the issue that Denis pointed out in the chat last night - that there really should be two steps at 7.